addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -4819,6 +4923,266 @@ class SnippetsMenu extends Component {
             return this.props.trigger_up(ev);
         }
     }
+    /**
+     * @private
+     * @param {OdooEvent} ev
+     */
+    _onOpenAddSnippetDialog(ev) {
+        this._openAddSnippetDialog(ev.data.snippetGroup, ev.data.initialSnippetEl);
+    }
+    /**
+     * Open a dialog with previews of snippets to add to the page.
+     *
+     * @param {String} [snippetGroup=null]
+     * @param {HTMLElement} initialSnippetEl
+     * @private
+     */
+    async _openAddSnippetDialog(snippetGroup = null, initialSnippetEl) {
+        this._mutex.exec(async () => {
+            let hookEl = null;
+            let dropZoneEls = null;
+            let isSnippetChosen = false;
+            const snippetThumbnails = this.el.querySelectorAll(".oe_snippet_thumbnail");
+            const isSnippetGroupBlock = initialSnippetEl.matches(".oe_snippet[data-snippet-group]");
```

Comment:
Wouldn't `isSnippetGroupClicked` be a more descriptive name? Because you'd expect that dragging a group would also set `isSnippetGroupBlock` to true (but it doesn't).

Reply:
Indeed, thanks